VMware.CloudFoundation.CertificateManagement.psm1

```diff
@@ -1276,7 +1276,7 @@ Function Install-EsxiCertificate {
                     $esxCertificatePem = Get-Content $crtPath -Raw
                     Set-VIMachineCertificate -PemCertificate $esxCertificatePem -VMHost $esxiFqdn -ErrorAction Stop -Confirm:$false
                     $replacedHosts.Add($esxiFqdn)
-                    Set-EsxiConnectionState -esxiFqdn $esxiFqdn -state "Disconnected" -timeout $timeout
+                    #Set-EsxiConnectionState -esxiFqdn $esxiFqdn -state "Disconnected" -timeout $timeout
```

Comment:
@bhumitra - is there a reason that this was commented out here?

Reply:
It should be uncommented. This is also outdated now. Closing it.